data_processing/common/Container.py

```diff
@@ -285,6 +285,8 @@ def get(self, type, name):
                 node.path = pathlib.Path(path.split(":")[-1])
             else:
                 node.path = pathlib.Path(path)
+            
+            node.static_path = str(node.path)
```

Comment:
Add 2 get methods to retrieve string path or PosixPath, and update code. To be addressed in a separate card.